### ISSUE TITLE
Import isn't working, nothing happens

### ISSUE THREAD

**Issue by gendalv** [sienori/Tab-Session-Manager#1164]
I wanted to export to batch-edit links as text, then import, but nothing happens, when trying to import: 
Import sessions->browse button->import button->message "sessions imported" , but nothing is added

## Platform information
* Platform (OS):  w11
* Version of browser: ff110
* Version of Tab Session Manager: 6.12.1

**Comment by gendalv** [comment]
the manual link import still works tho, so if you edit the file with regex using np++ to only keep the list of links you can import.

**Comment by cyphix333** [comment]
Thanks for the idea. :)

**Comment by badrelmers** [comment]
in fact it works.  you are for sure testing the export  import function so you exported the session then you imported them and you saw nothing changing, here i will explain why:
when you import a session that exist already , it will not be imported.
to force the import you have to change to "id" in the json file (at the end of the file):
 
![image](https://user-images.githubusercontent.com/40046489/234337676-1af6dad9-697b-4f02-ac4d-de38cedc1f61.png)
so i changed 
"id": "298bc79b-47ff-4d33-86f4-f4ca1417d845"
to 
"id": "298bc79b-47ff-4d33-86f4-f4ca1417d846"
and now it import the session

thank you sienori for this excellent extension

**Comment by cyphix333** [comment]
Nope, still not working for me after changing ID. Unless the dates in the file are affecting where it shows up in the session list after importing it?

**Comment by www-wagner** [comment]
I had the same issue on Vivaldi. I removed (not disabled, **removed**) and installed the extension again, import the settings (previously exported before removing the extension). Then I tried to import the sessions again and everything have worked fine.
